lib/delta.m

```diff
@@ -0,0 +1,5 @@
+function [y]=delta(t) 
+    y=zeros(1,length(t)); 
+    [~,i]=min(abs(t)); 
+    y(i)=1/(t(2)-t(1)); 
+end
```

Comment:
please add an end-of-line char at the end of the file (enter button), otherwise github complains